apps/condo/pages/callRecord/index.tsx

```diff
@@ -266,3 +274,20 @@ const CallRecordsPage: ICallRecordIndexPage = () => {
 CallRecordsPage.requiredAccess = CallRecordReadPermissionRequired
 
 export default CallRecordsPage
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+    const { req, res } = context
+
+    // @ts-ignore In Next 9 the types (only!) do not match the expected types
+    const { headers } = prepareSSRContext(req, res)
+    const client = initializeApollo({ headers })
+
+    const { redirect, user } = await prefetchAuthOrRedirect(client, context)
+    if (redirect) return redirect
+
+    const { activeEmployee } = await prefetchOrganizationEmployee({ client, context, userId: user.id })
```

Comment:
```suggestion
    await prefetchOrganizationEmployee({ client, context, userId: user.id })
```

and in other places where we not use `activeEmployee`

Reply:
I left it because we will use it later.
I can remove it now, but next pr we will add it again :)

Reply:
Removed unused variables

Reply:
same in cc